site/styles/_footer.scss

```diff
@@ -23,10 +23,14 @@ footer {
   margin-bottom: 30px;
 
   @include media-breakpoint-up(xl) {
-    margin: 0 50px 20px;
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
```

Comment:
This `justify-content` line doesn't appear to do anything because the child elements (nav ul and social icons div) are automatically filling all available space.  Can probably just delete it!